### ISSUE TITLE
CSD Validation ignores an error which is shown in CM-logs

### ISSUE THREAD

**Issue by kamir** [cloudera/cm_ext#7]
My CSD file is valid according to the local test. But the CM-logs show:

*2015-06-13 11:21:01,884 INFO main:com.cloudera.cmf.cdhclient.CdhExecutorFactory: Adding options to bad options list: 
2015-06-13 11:21:09,249 WARN main:com.cloudera.csd.components.CsdLocalRepository: Skipping [/opt/cloudera/csd/TTFAQ-1.1.0.jar]: { The service version [1.0] must match [1.1.0] for file [TTFAQ-1.1.0] }*

I had this issue once, but if you do not do CSD related you might miss this detail.

**Maybe we should not only validate the service.dsl file, but also the final JAR?**

**Comment by philipl** [comment]
Yeah, it's a missing feature. Right now it only validates the individual sdl file, but we could enhance it to look at a whole jar, like we do for whole parcels.
